src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -87,15 +87,15 @@
 											<!-- iOS workaround #361 DoubleAnimationUsingKeyFrames having a small delay (0.5 seconds) before animating -->
```

Comment:
here there's a comment for the BeginTime, not sure if it applies to other begin times.